llama_index/embeddings/bedrock.py

```diff
@@ -27,10 +27,10 @@ class Models(str, Enum):
 
 PROVIDER_SPECIFIC_IDENTIFIERS = {
     PROVIDERS.AMAZON.value: {
-        "embeddings": "embedding",
+        "get_embeddings_func": lambda r: r.get("embedding"),
     },
     PROVIDERS.COHERE.value: {
-        "embeddings": "embeddings",
+        "get_embeddings_func": lambda r: r.get("embeddings")[0],
```

Comment:
clever! But also... very annoying that bedrock doesn't return a consistent format...